hailjwt/hailjwt/hailjwt.py

```diff
@@ -0,0 +1,23 @@
+import jwt
+
+
+class JWTClient:
+    __ALGORITHM = 'HS256'
+
+    def __init__(self, secret_key):
+        self.secret_key = secret_key
```

Comment:
You should assert that the secret_key is at least 256-bits in length.

"A key of the same size as the hash output (for instance, 256 bits for "HS256") or larger MUST be used with this algorithm."

https://tools.ietf.org/html/rfc7518#section-3.2

I think you should have a class method to generate the secret key using the `secrets` package, see my other comment.

Reply:
I think the secret key should generally come from outside the application, right?

Reply:
I'm suggesting we have a utility for generating that key.

Reply:
done